src/strategies/erc721-balance-erc20/index.ts

```diff
@@ -0,0 +1,44 @@
+import { formatUnits } from '@ethersproject/units';
+import { call, multicall } from '../../utils';
+
+export const author = '0xAurelius';
+export const version = '0.0.1';
+
+const abi = [
+  'function balanceOf(address account) external view returns (uint256)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const rawERC20Balance = await call(
+    provider,
+    abi,
+    [options.erc20Address, 'balanceOf', [options.address]],
+    { blockTag }
+  );
+  const erc20Balance = parseFloat(formatUnits(
+    rawERC20Balance, options.erc20Decimals
+  ));
+
+  const response = await multicall(
+    network,
+    provider,
+    abi,
+    addresses.map((address: any) => [options.address, 'balanceOf', [address]]),
+    { blockTag }
+  );
+  return Object.fromEntries(
+    response.map((value, i) => [
+      addresses[i],
+      parseFloat(formatUnits(value.toString(), 0)) * erc20Balance
```

Comment:
Hey, @0xAurelius I think you can use the `math` strategy here and multiply operation,

Try passing erc721 and contract call into it, there was a similar strategy here https://github.com/snapshot-labs/snapshot-strategies/blob/master/src/strategies/erc20-balance-of-contract-multiplier/index.ts but no one use it,

Reply:
Ah ok thanks, I did look at that one but I couldn't figure out if it would do what I wanted

Will give it a shot 🤞